mmdet/models/dense_heads/vfnet_head.py

```diff
@@ -155,6 +155,11 @@ def __init__(self,
             sampler_cfg = dict(type='PseudoSampler')
             self.sampler = build_sampler(sampler_cfg, context=self)
 
+        # in order to unify the get_bbox logic. Not needed during training.
+        self.prior_generator = MlvlPointGenerator(
+            anchor_generator['strides'],
+            self.anchor_center_offset if self.use_atss else 0.5)
```

Comment:
what is `self.use_atss` used for

Reply:
He supports two target calculation modes: fcos and atss.

Reply:
Can we remove this after refactor the atss